Makefile

```diff
@@ -19,6 +19,10 @@ project:
 	@$(call check_for_gem,xcodeproj)
 	ruby Tests/Scripts/add_json_files.rb
 
+sourcery:
+	brew list sourcery || brew install sourcery || { echo "ERROR: Failed to locate or install sourcery; please install yourself"; exit 1; }
```

Comment:
do we need to do the checks using `brew` here? In at least the travis case you will be downloading this from github, not using brew, and then looking for it using brew. Let's just say `sourcery` is a requirement for the `sourcery` make target, and cut the error message out (it should be obvious if it tries to run and it doesn't exist)

Reply:
the travis case doesn't use `make sourcery`, because we download sourcery into a temporary directory and need to call it with the full path (see [here](https://github.com/mongodb/mongo-swift-driver/pull/213/files#diff-354f30a63fb0907d4ad57269548329e3R12)). 

so this only applies if you are doing this locally, in which case I assume all of us will be using brew-installed sourcery.
that said, the error is probably obvious enough, and we aren't performing a similar check for swiftlint, so I will remove this.